tools/dfx-orbit/src/canister.rs

```diff
@@ -31,7 +31,7 @@ pub enum RequestCanisterActionArgs {
     /// Request to call a canister method
     Call(RequestCanisterCallArgs),
     // Update a canister's settings (i.e its controller, compute allocation, or memory allocation.)
-    //UpdateSettings(RequestCanisterUpdateSettingsArgs),
```

Comment:
Why was this commented out before?

Reply:
As written above, I did not surface this code to the CLI on purpose, because Kepler told me that there would be a breaking change.